pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-pediatric-invitation.html

```diff
@@ -84,20 +84,20 @@
         <td style="padding: 50px 0 0 0;">
             <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Dear -ddp.proxy.firstName-,</p>
 
-            <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Thank you for being part of the Leiomyosarcoma Project. As a part of the project, we are partnering with Invitae in order to share information with participants about their germline DNA (or “normal” DNA).</p>
+            <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Thank you for being part of the Leiomyosarcoma Project and for sharing your child’s saliva and/or blood samples with us. We know it takes extra time and energy to send us your child’s samples, and we appreciate your efforts. We have also received a tumor sample from a hospital where your child received care. You may remember from the initial consent that as a part of the project, you can now decide if you’d like to learn more about your child’s germline DNA (or “normal” DNA) by having genetic counseling and germline genetic testing through a partnership with Genome Medical and Invitae. The information from testing may include details that inform cancer risk for you and your family.</p>
 
-            <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">We are now asking you to opt in if you would like to take part in this additional aspect of the study. Taking part in this is completely optional and will not have any impact on your child’s overall participation in the LMSproject.</p>
+            <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">You will need to opt in if you would like to take part in this additional aspect of the study. Taking part in this is completely optional and will not have any impact on your child’s overall participation in the LMSproject.</p>
 
             <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">The following steps are involved with participation in this additional aspect of the study:</p>
 
             <ul style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">
                 <li>Sign an additional consent form</li>
-                <li>Schedule and have a telehealth appointment with a genetic counselor from Invitae or their telehealth partner, Genome Medical</li>
+                <li>Schedule and have a phone appointment with a genetic counselor from Invitae or their telehealth partner, Genome Medical</li>
```

Comment:
should be "Schedule and have a phone appointment with a genetic counselor from Genome Medical" as per source